apps/crn-frontend/src/shared-research/state.ts

```diff
@@ -160,3 +161,19 @@ export const useSetResearchOutputItem = () => {
     setRefresh(refresh + 1);
   };
 };
+
+export const useCanUpdateWorkingGroupResearchOutput = (
```

Comment:
This isn't really state as it doesn't touch recoil. Why do we have this function twice with the one thats currently in RO. (( Self: Come back and figure this out better ))